src/Http/Livewire/LivewireDatatable.php

```diff
@@ -698,7 +698,13 @@ public function addGlobalSearch()
                     $this->searchableColumns()->each(function ($column, $i) use ($query, $search) {
                         $query->orWhere(function ($query) use ($i, $search) {
                             foreach ($this->getColumnField($i) as $column) {
-                                $query->orWhereRaw('LOWER('.$column.') like ?', "%$search%");
+                                if (is_array($column)) {
```

Comment:
I would like to suggest the use of `when` on the query.
See [reference](https://laravel.com/docs/6.x/queries#conditional-clauses) for more information.